src/lib/api/base.js

```diff
@@ -11,7 +11,7 @@ const apiConfig = {
 };
 const http = axios.create(apiConfig);
 
-const HTTP_STATUS_CODES_WITH_ERROR_PAGE = [404, 429, 500];
+const HTTP_STATUS_CODES_WITH_ERROR_PAGE = [400, 404, 429, 500];
```

Comment:
I am not 100% sure about this: if I am not mistaken, when there is a 400 error from the backend, it shows a banner on the top right.
To test it, you should try, for example, to check-out a loan that is in "completed" state and check the response code.